app/src/main/java/com/ozsaat/feeddit/MainActivity.java

```diff
@@ -32,16 +28,16 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
 
-    public ArrayList<String> getFeedlyItems() {
-        ArrayList<String> items = new ArrayList<>();
+    public ArrayList<NavigationItem> getFeedlyItems() {
+        ArrayList<NavigationItem> items = new ArrayList<>();
         items.add("S1-A");
```

Comment:
Here you should be getting errors. Because you are trying to add a ```String``` into an ```ArrayList<NavigationItem>```. Same for next lines and same for ```getRedditItems()``` method